rules_gapic/bazel/src/main/java/com/google/api/codegen/bazel/ApiVersionedDir.java

```diff
@@ -40,6 +44,18 @@
 
   private static String CLOUD_AUTH_SCOPE = "https://www.googleapis.com/auth/cloud-platform";
 
+  private static final String[] PRESERVED_PROTO_LIBRARY_STRING_ATTRIBUTES = {
```

Comment:
How hard would it be to make this stuff configurable (probably to the extent of passing it from command line)?

Reply:
I would prefer not to do it now until we have the first user who needs it. That's a kind of YAGNI :)